### PR TITLE
Add okteto channel

### PR DESCRIPTION
**Pull request by pchico83**
`okteto` is a tool to develop directly in your dev Kubernetes cluster. No commit, build or push required. The channel will be used to share experiences between okteto users.

https://github.com/okteto/okteto

I'm @pablo on k8s.slack, happy to answer any questions :)

### CODE REVIEW

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -212,6 +212,7 @@ channels:
   - name: norw-users
   - name: office-hours
   - name: opencontainers
+  - name: okteto
```
Comment (idealhack) [inline communication/slack-config/channels.yaml]: Could you move it above by one line? otherwise lgtm

Comment (pchico83) [reply communication/slack-config/channels.yaml]: Right. Done!

**Pull request merged**